### ISSUE TITLE
Pop up "Marble Moments Crashed"

### ISSUE THREAD

**Issue by lisacarree** [marblear/issues#2]
After I close the app, I get this pop up (see picture) every time. 
![IMG_9247 2](https://user-images.githubusercontent.com/81170702/112001037-d7c19580-8b1e-11eb-862b-f4c5cde275e7.PNG)

**Comment by derwaldgeist** [comment]
Thanks. This is a known behavior if the app is being started from TestFlight. It does not happen in the production version, though. Since the crash report never contained any data, I never found out why this happens. Could you please tap on "Share"? Maybe your device is different and sends some valid data.

**Comment by derwaldgeist** [comment]
I investigated this problem today, for a couple of hours.

I could identify the internal iOS error message that causes the crash message to pop up. It is because iOS thinks the app hasn't shut down gracefully within 10 seconds (this is also the reason why the popup appears only after a while). However, I could not find any particular reason why this is actually being triggered.

I found similar reports on the internet, but none of the solutions that were suggested really worked. Fortunately, the error message only shows up if the app is being started from TestFlight, so regular users in the App Store won't see it. The only drawback I have noticed is that a subsequent app start will be delayed by 10s max., because iOS thinks the app is still running.

Maybe it's a bug inside Unity that prevents app teardown. Thus I posted a question on their forums.
